assets/queries/cloudFormation/elastic_search_domain_without_encryption_at_rest_options/query.rego

```diff
@@ -0,0 +1,40 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document
+  resource = document[i].Resources[name]
+  resource.Type == "AWS::Elasticsearch::Domain"
+  properties := resource.Properties
+  
+  exists_encryptionAtRestOptions := object.get(properties, "EncryptionAtRestOptions", "undefined") != "undefined"
+  not exists_encryptionAtRestOptions
```

Comment:
Simplify lines 9, 10 to just:

`object.get(properties, "EncryptionAtRestOptions", "undefined") == "undefined"`